src/app/backend/resource/logs/logs.go

```diff
@@ -46,60 +45,71 @@ var OldestLogLineId = LogLineId{
 
 // Default log view selector that is used in case of invalid request
 // Downloads newest DefaultDisplayNumLogLines lines.
-var DefaultLogViewSelector = &LogViewSelector{
-	RelativeFrom:       1 - DefaultDisplayNumLogLines,
-	RelativeTo:         1,
-	ReferenceLogLineId: NewestLogLineId,
+var DefaultSelection = &Selection{
+	OffsetFrom:     1 - DefaultDisplayNumLogLines,
+	OffsetTo:       1,
+	ReferencePoint: NewestLogLineId,
 }
 
-// AllLogViewSelector returns all logs.
-var AllLogViewSelector = &LogViewSelector{
-	RelativeFrom:       -MaxLogLines,
-	RelativeTo:         MaxLogLines,
-	ReferenceLogLineId: NewestLogLineId,
+// Returns all logs.
+var AllSelection = &Selection{
+	OffsetFrom:     -MaxLogLines,
+	OffsetTo:       MaxLogLines,
+	ReferencePoint: NewestLogLineId,
 }
 
-// LogViewSelector selects a slice of logs.
-// It works just like normal slicing, but indices are referenced relatively to certain reference line.
-// So for example if reference line has index n and we want to download first 10 elements in array we have to use
-// from -n to -n+10. Setting ReferenceLogLineId the first line will result in standard slicing.
-type LogViewSelector struct {
-	// ReferenceLogLineId is the ID of a line which should serve as a reference point for this selector.
-	// You can set it to last or first line if needed. Setting to the first line will result in standard slicing.
-	ReferenceLogLineId LogLineId
-	// First index of the slice relatively to the reference line(this one will be included).
-	RelativeFrom int
-	// Last index of the slice relatively to the reference line (this one will not be included).
-	RelativeTo int
+// Representation of one page of logs
+type LogPage struct {
```

Comment:
I was thinking about trying to make it consistent with other resources. 

Typically we name things *list or *details (podlist, poddetails). I was wondering if the following structure would make sense (or lets say indicates how it is to be used):

```
LogDetails {
     LogInfo (LogMeta?)
     LogSelection
     LogLines
} 
```